about.html

```diff
@@ -0,0 +1,128 @@
+
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/205459598-a68b758d-0d4e-48b4-a035-a803f0761ed3.PNG)

**[Optional]**

- [ ] As you can see in the image above in your mobile about page you have a white spacing at the top of your page 😨 KIndly remove it to make your project look similar to the design which will make your project look professional 😊

Reply:
ok , thank you i will make it.